x-pack/plugins/reporting/server/lib/event_logger/types.ts

```diff
@@ -8,19 +8,21 @@
 import { LogMeta } from 'src/core/server';
 import { ActionType } from './';
 
-interface ActionBase<A extends ActionType> {
+export interface ReportingAction<A extends ActionType> extends LogMeta {
   event: {
     timezone: string;
   };
   message: string;
   kibana: {
     reporting: {
-      actionType?: A;
+      actionType: A;
       id?: string; // "immediate download" exports have no ID
       jobType: string;
       byteSize?: number;
+      csvRows?: number;
```

Comment:
I really wonder if this should be:
```
csv?: { rows: number; };
```

We could have more CSV-specific metrics: number of columns in the CSV is one. But if there are other settings that affect performance, we should think of adding them as well.